views/layout.php

```diff
@@ -0,0 +1,32 @@
+
+<DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" href="../../assets/css/bootstrap.min.css">
+    <link rel="stylesheet" href="../../assets/css/custom.css">
+    <script src="../../assets/js/main.js"></script>
```

Comment:
Why this script tag here is bad in terms of performance?

Reply:
This script is used in all pages.. so I thought to insert here..